codalab/lib/metadata_util.py

```diff
@@ -94,13 +94,14 @@ def parse_metadata_form(bundle_subclass, form_result):
             if metadata_key not in metadata_types:
                 raise UsageError('Unexpected metadata key: %s' % (metadata_key,))
             metadata_type = metadata_types[metadata_key]
+
             if metadata_type == list:
-                if any(unicode_util.contains_unicode(v) for v in result[metadata_key]):
+                remainders = remainder.split() if remainder else []
+                if any(unicode_util.contains_unicode(r) for r in remainders):
                     raise UsageError(
-                        'Metadata cannot contain unicode: %s = %s'
-                        % (metadata_key, result[metadata_key])
+                        'Metadata cannot contain unicode: %s = %s' % (metadata_key, remainder)
                     )
-                result[metadata_key] = remainder.split() if remainder else []
+                result[metadata_key] = remainders
```

Comment:
Not exactly sure if I have made the right changes here. From the context, it looks like we want to check for each element of the remainder to see if any of it contains Unicode character. But let me know if this is not the case.